### ISSUE TITLE
Run estimations of commercial catch data derived from RDBES in TAF

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-taf/doc#266]

**Comment by neil-ices-dk** [comment]
To be discussed; was talked about by Jens (DIG), David (SC-RDB) and Kjems and Neil; There is a general positive feeling that this would be a good way forward, although Kjems will need a bit more help in getting his head into how TAF works and how it handles versioning etc.

**Comment by Lotte-ices-dk** [comment]
Interesting - how would this link to the estimations done in the RDBES? Assume this is to align the processes between EU and non-EU ICES MCs? All in favor for exploring.

**Comment by neil-ices-dk** [comment]
Ah, clearly my grammar/plain English has let me down. The idea being to not do the estimates in RDBES but in TAF based on RDBES data.

**Comment by Lotte-ices-dk** [comment]
Or me being stupid and unable to understand written English? :) Hrm, okay, let's discuss and find out how to do this in a way that we're not upsetting the entire RDBES community (again)

**Comment by arnima-github** [comment]
Maybe it would be good to have a TAF workshop with the Data Centre in the near future.
Colin and I could cover topics like:
1. Practical overview of R scripts and the R language
2. The TAF workflow, bringing in data and software, scripting any analysis
3. Using TAF to version datasets
4. Other relevant aspects of TAF for the Data Centre

**Comment by neil-ices-dk** [comment]
great idea - you just let me know when and i will provide refreshments!

**Comment by arnima-github** [comment]
Colin mentioned we might do a mini-workshop with the RDBES team (Kjems, Ejaz, Kadji) instead of all the Data Centre.

We're probably aiming for a long-term collaboration between the TAF team and the RDBES team, and we could have the first technical meeting(s) in a mini-workshop format, covering topics 1-4 above with the RDBES workflow in mind.

**Comment by neil-ices-dk** [comment]
makes sense, bear in mind you will have a parallel collaboration with acoustics, so up to you if this makes sense to keep separate or take them in one go
